cmd/cluster.go

```diff
@@ -12,13 +12,18 @@ func setupClusterCommand() *cobra.Command {
 		Use:   "up",
 		Short: "Boot up the testing cluster",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := cluster.BootUp()
+			d, err := cmd.Flags().GetBool("d")
+			if err != nil {
+				return err
+			}
+			err = cluster.BootUp(d)
 			if err != nil {
 				return errors.Wrap(err, "booting up the cluster failed")
 			}
 			return nil
 		},
 	}
+	upCommand.Flags().Bool("d", false, "Run cluster as daemon")
```

Comment:
@mtojek This gets `--d` working but not `-d`. I assume there is a better way to define this in cobra?

Reply:
Yes, here is a sample with short flag: https://github.com/elastic/elastic-package/blob/master/cmd/testrunner.go#L28

You can add both forms (daemon/d).

Reply:
Hm, now with using `PersistentFlags().BoolP` it stops working completely.

Reply:
Sorry, I meant to use the method `BoolP`, but with `Flags()`. Persistent flags are reserved rather for general purpose flags, independent from commands, e.g. verbose, debug, etc.